src/main/java/org/drtshock/Fries.java

```diff
@@ -0,0 +1,20 @@
+package org.drtshock;
+
+import org.drtshock.exceptions.BurntException;
+import org.drtshock.exceptions.NotDeliciousException;
+import org.drtshock.types.NotDeliciousReason;
+
+public class Fries extends Potato {
+
+	public Fries(boolean isVegan) {
```

Comment:
the potato hates tabs

Reply:
Understood, if Potato hates tabs, everything shall become spaces.